core/build.sbt

```diff
@@ -1,13 +1,12 @@
-
 resolvers ++= Seq(
   "scalaz.bintray" at "http://dl.bintray.com/scalaz/releases",
   "oncue.bintray" at "http://dl.bintray.com/oncue/releases"
 )
 
 libraryDependencies ++= {
   val ermineVersion =
-    if(scalazStreamVersion.value.startsWith("0.7")) "0.3.3a"
-    else "0.3.3"
+    if(List("0.7", "0.8").find(prefix => scalazStreamVersion.value.startsWith(prefix)).isDefined) "0.3.4a"
```

Comment:
I'm not sure this is right. This will pull in the wrong version of scalaz... given there are only 0.7 and 0.8 builds, the `else` clause will never be reached. I think this can be taken out.